src/main/java/org/takes/facets/auth/social/PsGithub.java

```diff
@@ -100,10 +118,13 @@ public Response exit(final Response response,
      * @return The user found in Github
      * @throws IOException If fails
      */
-    private static Identity fetch(final String token) throws IOException {
-        final String uri = new Href("https://api.github.com/user")
-            .with("access_token", token)
-            .toString();
+    private Identity fetch(final String token) throws IOException {
+        final String uri = new Href(
+            String.format(
+                "%s/user",
```

Comment:
@dmzaytsev This is a confusing design. Why are we changing the Github URI? Instead let's pass a fake request that we can check as originally described in #42.